### PR TITLE
Fixing mb bugfest and clean deployment schema difference and version …

### PR DESCRIPTION
**Pull request by rmathew1011**
[MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 24.0.4 2022-08-11
```
Comment (marcjohnson-kint) [inline NEWS.md:1]: This version hasn't been released yet, so it should not have a release date

```suggestion
## 24.0.4 IN-PROGRESS
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+* Fixing mb bugfest and clean deployment schema difference version modified ([MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935))
```
Comment (marcjohnson-kint) [inline NEWS.md:3]: Given that this is documentation for an operator or user of the module, I think it is better to use language that states what has happened.

```suggestion
* Adds integrity checks for statistical code types during upgrade ([MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935))
```

File: src/main/resources/templates/db_scripts/schema.json [hunk 3]
```diff
@@ -1015,7 +1015,7 @@
     {
       "run": "after",
       "snippetPath": "statisticalCodeTypesReferenceCheckTrigger.sql",
-      "fromModuleVersion": "23.0.0"
+      "fromModuleVersion": "24.0.4"
```
Comment (marcjohnson-kint) [inline src/main/resources/templates/db_scripts/schema.json:1018]: The next targeted release on the 24.0.x branch is 24.0.3. Why was 24.0.4 chosen for the version where the script should run?

File: NEWS.md (hunk 1 above)
Comment (rmathew1011) [reply NEWS.md:1]: Text modified

File: NEWS.md (hunk 2 above)
Comment (rmathew1011) [reply NEWS.md:3]: Description updated

File: src/main/resources/templates/db_scripts/schema.json (hunk 3 above)
Comment (rmathew1011) [reply src/main/resources/templates/db_scripts/schema.json:1018]: updated to 24.0.3

**Review by marcjohnson-kint — APPROVED** [review]
This seems ok to me. It would be ideal if the upgrade (from the lotus version to this branch) could be tested locally to check it works as expected.

**Review by jeremythuff — APPROVED** [review]

**Pull request merged**